src/common/preferences/preferences.html

```diff
@@ -104,6 +102,19 @@
 				</div>
 			</div>
 			<div class="group">
+				<div class="group-title">Report Errors</div>
+				<div class="group-content">
+					<div id="general-no-errors">
+						Zotero Connector has not experienced any errors since it was first started.
+					</div>
+					<div id="general-have-errors" style="display:none">
+						<p>Zotero Connector has experienced the following errors since it was first started:</p>
```

Comment:
Maybe `The following errors have occurred since the Zotero Connector was started:`